src/libs/EmojiUtils.js

```diff
@@ -186,9 +186,10 @@ function addToFrequentlyUsedEmojis(frequentlyUsedEmojis, newEmoji) {
 /**
  * Replace any emoji name in a text with the emoji icon
  * @param {String} text
+ * @param {Boolean} addSpace
  * @returns {String}
  */
-function replaceEmojis(text) {
+function replaceEmojis(text, addSpace = false) {
```

Comment:
I think we should rename this parameter to `isSmallScreenWidth` for more clarity. As in the tests below passing `addSpace` as true when there's text after it is misleading, another engineer might assume that we always add space if that parameter is true.